config/base/webhook/manifests.yaml

```diff
@@ -11,7 +11,7 @@ webhooks:
     service:
       name: webhook-service
       namespace: system
-      path: /validate
+      path: /validate-kafka.banzaicloud-io-v1alpha1-kafkatopic
```

Comment:
**Question**: wan't this intended to be `kafka-banzaicloud` like it is for the `kafkacluster` validation endpoint?

```suggestion
      path: /validate-kafka-banzaicloud-io-v1alpha1-kafkatopic
```

Reply:
You should regenerate the manifests to fix this based on the fixed path in the code.

Reply:
This is still an issue in the latest state.